rollup.config.next.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
+import fs from 'fs';
+import base from './rollup.config';
+
+let pkg = JSON.parse(fs.readFileSync('./package.json'));
+
+export default Object.assign(base, {
+	dest: pkg['jsnext:main'],
+	sourceMap: path.resolve(pkg['jsnext:main']),
```

Comment:
Also you may use `targets` option which is an array of different `write` options.